tox.ini

```diff
@@ -36,9 +37,7 @@ commands =
     {envpython} -m coverage report
 
 [testenv:flake8]
-deps =
-    -rrequirements.flake8.txt
-skip_install = true
```

Comment:
The `skip_install` directive seemed to be causing issues for me, so I removed it. The flake8 requirements are now moved to an "extra" group defined in `pyproject.toml`; the `requirements.flake8.txt` file should no longer be needed.

Reply:
I'm okay with this